pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -230,6 +230,10 @@ def tearDown(self):
         del self.G
         del self.G1
 
+    def test__eq__(self):
```

Comment:
@Vaibhavdixit02  Tests are failing because you have defined `test__eq__` method after `tearDown` method.

Reply:
I didn't pay attention, sorry! Fixed now.